pom.xml

```diff
@@ -60,23 +53,41 @@
 	</modules>
 
 	<properties>
-		<maven.compiler.source>1.8</maven.compiler.source>
-		<maven.compiler.target>1.8</maven.compiler.target>
-		<spring-cloud.version>2020.0.0-SNAPSHOT</spring-cloud.version>
-		<spring-cloud-sleuth.version>2.2.4.BUILD-SNAPSHOT</spring-cloud-sleuth.version>
-		<zipkin-gcp.version>0.17.0</zipkin-gcp.version>
 		<app-engine-maven-plugin.version>2.3.0</app-engine-maven-plugin.version>
-		<kotlin.version>1.3.31</kotlin.version>
+		<asciidoctor-maven-plugin.version>2.0.0</asciidoctor-maven-plugin.version>
+		<checkstyle.additional.suppressions.file>${maven.multiModuleProjectDirectory}/src/checkstyle/checkstyle-suppressions.xml</checkstyle.additional.suppressions.file>
```

Comment:
Can we try to group these checkstyle properties together with the other ones?
This is just so many properties. I think the idea was that they can be overwritten more easily in the child POMs. However, since we're not using inheritance anymore, the settings can probably just go straight into the plugin configuration.

Reply:
I alphabetized them for easier search+finding, but you're right they can probably go straight into the plugin config.

Reply:
Moved a few of these properties inline.